components/common/map/LocationSearch.tsx

```
@@ -0,0 +1,185 @@
+  const [isLoading, setIsLoading] = useState(false);
+
+  const searchLocation = async (query: string) => {
+    if (!query.trim()) {
+      setSearchResults([]);
+      return;
+    }
+
+    setIsLoading(true);
+    try {
+      const response = await fetch(
+        `https://nominatim.openstreetmap.org/search?format=json&q=${encodeURIComponent(query)}&limit=5`
```

ESLint

[error] Insert `,`

(prettier/prettier)